sys/zuse.hoon

```diff
@@ -6288,9 +6288,7 @@
       ;~(plug auri (punt ;~(pfix hax (cook crip (star pque)))))
     ::                                                  ::  ++auri:de-purl:html
     ++  auri                                            ::  2396 URL
-      %+  cook
-        |=  a/purl
-        ?.(?=(hoke r.p.a) a a(p.p &))
+      %+  cook  |~(a/purl a)
```

Comment:
If you're just producing the sample of the cook gate, you should be able to remove the cook entirely and leave just the parsing code below?

Reply:
Yup, just tried removing this line and it still worked. Thanks for the help :+1: